ci/win_download_deps.cmd

```diff
@@ -13,7 +13,8 @@ pwsh -command Invoke-WebRequest -MaximumRetryCount 4 -OutFile python-2.7.msi htt
 pwsh -command Invoke-WebRequest -MaximumRetryCount 4 -OutFile msmpisetup.exe https://download.microsoft.com/download/a/5/2/a5207ca5-1203-491a-8fb8-906fd68ae623/msmpisetup.exe || goto :error
 pwsh -command Invoke-WebRequest -MaximumRetryCount 4 -OutFile msmpisdk.msi https://download.microsoft.com/download/a/5/2/a5207ca5-1203-491a-8fb8-906fd68ae623/msmpisdk.msi || goto :error
 
-:: nsis plugin
+:: nsis + plugin
+curl -C - -LO http://prdownloads.sourceforge.net/nsis/nsis-3.05-setup.exe
```

Comment:
Should this not be the same PowerShell command as above? Why `curl`?

Reply:
Long story short, it can be very tricky with sourceforge (download link is not permanent).

Reply:
Ouch, OK